Routing/Loader/RestRouteLoader.php

```diff
@@ -162,7 +162,7 @@ private function getControllerLocator($controller)
      */
     protected function findClass($file)
     {
-        @trigger_error('The '.__METHOD__.' method is deprecated since version 1.8 and will be removed in 2.0. Use '.__NAMESPACE__.'\ClassUtils::findClassInFile() instead.', E_USER_DEPRECATED);
+        @trigger_error('The '.__METHOD__.' method is deprecated since version 1.8 and will be removed in 2.0.', E_USER_DEPRECATED);
```

Comment:
It is in the [symfony best practices](http://symfony.com/doc/current/contributing/code/standards.html#structure) so I think it is better to use it.